spec/models/article_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  subject { Article.new(title: "Title123", description: "Description12") }
```

Comment:
please create Article by using factory bot
https://medium.com/@lukepierotti/setting-up-rspec-and-factory-bot-3bb2153fb909

FactoryBot.define do